identity/cache_test.go

```diff
@@ -0,0 +1,48 @@
+package identity
+
+import (
+	"testing"
+	"github.com/mysterium/node/service_discovery/dto"
+	"github.com/stretchr/testify/assert"
+	"os"
+)
+
+var file = "/tmp/cache.json"
+
+func TestIdentityCache_StoreIdentity(t *testing.T) {
+	identity := dto.Identity("0x000000000000000000000000000000000000000A")
+	cache := identityCache{
+		File: file,
+	}
+
+	err := cache.StoreIdentity(&identity)
+	assert.Nil(t, err)
+}
+
+func Test_IdentityCacheGetIdentity(t *testing.T) {
+	identity := dto.Identity("0x000000000000000000000000000000000000000A")
+	cache := identityCache{
+		File: file,
+	}
+
+	err := cache.StoreIdentity(&identity)
+	assert.Nil(t, err)
+
+	_, err = os.Stat(file)
```

Comment:
Purpose of this line? err is ignored anyway.

Reply:
It's for a test, i don't need the file info, I just want to check if it exists